### PR TITLE
Addition of the slides for the BioBlend API presentation (Day 3, EGDW 2017 strasbourg)

### PR DESCRIPTION
**Pull request by odoppelt**

### CODE REVIEW

**Comment by bgruening** [comment]
@odoppelt wonderful!
Do you want to port it to the new template as well? We can help you? Do we have time to do this on Monday? ;)

File: Dev-Corner/slides/bioblend_api.html [hunk 1]
```diff
@@ -0,0 +1,806 @@
+<!DOCTYPE html>
```
Comment (bebatut) [inline Dev-Corner/slides/bioblend_api.html]: Can you put on the top:

```
---
layout: tutorial_slides
topic_name: "Dev-Corner"
tutorial_name: bioblend_api
logo: "GTN"
---
```

It is useful to find the good templates.  Like this, you do not need any html in the top. 

Thanks

File: Dev-Corner/tutorials/bioblend_api.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+BioBlend API
```
Comment (bebatut) [inline Dev-Corner/tutorials/bioblend_api.md]: Can you remove this file if you are not filling it? Thanks

File: metadata/Dev-Corner.yml [hunk 3]
```diff
@@ -152,6 +152,24 @@ material:
...
+    zenodo_link: ""
+    galaxy_tour: ""
+    hands_on: "no"
+    slides: "yes"
+
+    objectives:
+    - "Get familiar with BioBlend with ipython"
+    - "Launch a Galaxy job / Visualize your actions with Galaxy"
+    - "Launch a Galaxy workflow / Visualize your actions with Galaxy"
+
```
Comment (bebatut) [inline metadata/Dev-Corner.yml]: Can you add the questions and the key_points, please? They are part of the templating and will be empty otherwise. Thanks

Comment (odoppelt) [inline metadata/Dev-Corner.yml]: done

File: Dev-Corner/slides/bioblend_api.html (hunk 1 above)
Comment (odoppelt) [inline Dev-Corner/slides/bioblend_api.html]: we added this but used the base_slides layout.

File: Dev-Corner/slides/index.html [hunk 4]
```diff
@@ -81,6 +81,7 @@
 
                         - [Tool development and integration into Galaxy](tool_integration.html)
                         - [Data source integration](data_source_integration.html)
+			- [BioBlend API](bioblend_api.html)
```
Comment (nsoranzo) [inline Dev-Corner/slides/index.html]: This is not aligned correctly because it is using tabs instead of spaces.

Comment (odoppelt) [inline Dev-Corner/slides/index.html]: Hello Nicola,
I replaced the tabs.
Cheers,
Olivia

File: Dev-Corner/slides/bioblend_api.html [hunk 5]
```diff
@@ -148,8 +148,8 @@
 
 * To resume, in this first part, we need to retrieve:
   1. A *history_id*, where the input data is and where the output data will be
-  2. A *tool_id*, which will tell Galaxy which tool to execute
-  3. *tool_inputs*, dictionary storing the data used to run the tool
+  2. A *tool_id*, which wills tell Galaxy which tool to execute
```
Comment (nsoranzo) [inline Dev-Corner/slides/bioblend_api.html]: Actually `will` was the correct form.

File: metadata/Dev-Corner.yml [hunk 6]
```diff
@@ -152,6 +152,39 @@ material:
...
+    type: "tutorial"
+    name: "bioblend_api"
+    zenodo_link: ""
+    galaxy_tour: ""
+    hands_on: "yes"
+    slides: "yes"
+
+    questions:
+    - "What is the use of BioBlend?"
+    - "How to use Galaxy through command lines?"
```
Comment (nsoranzo) [inline metadata/Dev-Corner.yml]: s/command lines/the command line/

File: metadata/Dev-Corner.yml [hunk 7]
```diff
@@ -152,6 +152,39 @@ material:
     - "Requires understanding of both the Galaxy ecosystem as well as HTML5/JS"
     - "Performance is more important than for static Galaxy tools"
 
+-
+    title: "BioBlend module, a python library to use Galaxy API"
```
Comment (nsoranzo) [inline metadata/Dev-Corner.yml]: s/python/Python/ and s/use Galaxy/use the Galaxy/

File: metadata/Dev-Corner.yml [hunk 8]
```diff
@@ -152,6 +152,39 @@ material:
...
+    galaxy_tour: ""
+    hands_on: "yes"
+    slides: "yes"
+
+    questions:
+    - "What is the use of BioBlend?"
+    - "How to use Galaxy through command lines?"
+
+    objectives:
+    - "Get familiar with BioBlend using ipython"
```
Comment (nsoranzo) [inline metadata/Dev-Corner.yml]: s/ipython/IPython/

File: metadata/Dev-Corner.yml [hunk 9]
```diff
@@ -152,6 +152,39 @@ material:
...
+    - "Launch a Galaxy job / Visualize your actions with Galaxy"
+    - "Launch a Galaxy workflow / Visualize your actions with Galaxy"
+
+    requirements:
+    -
+        title: "ipython"
+        type: "external"
+        link: ""
```
Comment (nsoranzo) [inline metadata/Dev-Corner.yml]: https://ipython.org/ ?

File: metadata/Dev-Corner.yml [hunk 10]
```diff
@@ -152,6 +152,39 @@ material:
...
+    - "How to use Galaxy through command lines?"
+
+    objectives:
+    - "Get familiar with BioBlend using ipython"
+    - "Launch a Galaxy job / Visualize your actions with Galaxy"
+    - "Launch a Galaxy workflow / Visualize your actions with Galaxy"
+
+    requirements:
+    -
+        title: "ipython"
```
Comment (nsoranzo) [inline metadata/Dev-Corner.yml]: s/ipython/IPython/

File: metadata/Dev-Corner.yml [hunk 11]
```diff
@@ -152,6 +152,39 @@ material:
...
+        type: "external"
+        link: ""
+
+    time_estimation: "2h"
+
+    key_points:
+    - "Play with ipython"
```
Comment (nsoranzo) [inline metadata/Dev-Corner.yml]: s/ipython/IPython/

**Comment by bgruening** [comment]
:tada: :balloon:

**Pull request merged**